packages/api-codec/src/Metadata.ts

```diff
@@ -283,11 +283,6 @@ export default class RuntimeMetadata extends Struct {
     ) as RuntimeMetadata;
   }
 
-  // FIXME Really not crazy about having to manually add all the getters. Preferably it should
-  // be done automagically in the actual Struct - however what is really important here
-  // here is that we should nbot lose the autocompletion and checking that TS gives us. So if
-  // we have to choose between the 2, manual defs it would have to be.
-
```

Comment:
Well, that boat has sailed (for now). Added it mostly everywhere we use structs. It is not _that_ bad.